### PR TITLE
Prometheus 2.1

### PR DESCRIPTION
**Pull request by goruha**
## what
* Implement Prometheus umbrella chart

## why
* Comprehensive, extensible monitoring framework

### CODE REVIEW

**Comment by osterman** [comment]
@goruha abandon this PR now since we have the others?

**Comment by goruha** [comment]
I'll fix it. And I do not want it to be merged yet.

File: incubator/kube-prometheus/templates/_configmaps.json.tpl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ "items": [
+  {
+    "key": "{{ .Release.Namespace }}/prometheus-{{ .Release.Name }}-rules",
+    "checksum": "33bbd11f3d6ecfa14af3d46c46e91de4906fe86d2429f34ba6ee74082f9d6414"
```
Comment (osterman) [inline incubator/kube-prometheus/templates/_configmaps.json.tpl]: Why is this `checksum` hardcoded?

Comment (osterman) [inline incubator/kube-prometheus/templates/_configmaps.json.tpl]: If we need to hardcode it, then how to calculate it?

Comment (goruha) [inline incubator/kube-prometheus/templates/_configmaps.json.tpl]: Done

File: incubator/prometheus/templates/_configmaps.json.tpl [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {{- if and .Values.rules.specifiedInValues .Values.rules.value }}
   {
     "key": "{{ .Release.Namespace }}/prometheus-{{ .Release.Name }}-rules",
-    "checksum": "mock_hash"
+    "checksum": "0000000000000000000000000000000000000000000000000000000000000000"
```
Comment (osterman) [inline incubator/prometheus/templates/_configmaps.json.tpl]: Did it need to be a certain length?

**Comment by osterman** [comment]
+2

File: incubator/prometheus/templates/_configmaps.json.tpl (hunk 2 above)
Comment (osterman) [inline incubator/prometheus/templates/_configmaps.json.tpl]: @goruha 
> just try to make it same length as original to reduce risk of bug

**Pull request merged**